app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+  def show_errors object, field_name
+    return unless object.errors.any?
+    if object.errors.messages[field_name].present?
+      [t("errors.user.#{field_name}") , object.errors.messages[field_name].join(", ")].join(" ")
```

Comment:
```suggestion
      [t("errors.user.#{field_name}"), object.errors.messages[field_name].join(", ")].join(" ")
```
trước dấu `,` k để space

Reply:
Dạ em sửa rồi ạ